### PR TITLE
Feat(Query): Networks Not Set for docker compose

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Networks Not Set for docker compose

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/dockerCompose/networks_not_set/metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "ce14a68b-1668-41a0-ab7d-facd9f784742",
+  "queryName": "Networks Not Set",
+  "severity": "MEDIUM",
+  "category": "Resource Management",
```
Comment (rafaela-soares) [inline assets/queries/dockerCompose/networks_not_set/metadata.json:5]:
```suggestion
  "category": "Networking and Firewall",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — DISMISSED** [review]
LGTM 🚀

File: assets/queries/dockerCompose/networks_not_set/query.rego [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+CxPolicy[result] {
+	resource := input.document[i]
+	service_parameters := resource.services[name]
+    not common_lib.valid_key(service_parameters, "networks")
+
+	result := {
+		"documentId": sprintf("%s", [resource.id]),
+		"searchKey": sprintf("services.%s",[name]),
+		"issueType": "IncorrectValue",
```
Comment (joaorufi) [inline assets/queries/dockerCompose/networks_not_set/query.rego:13]:
```suggestion
		"issueType": "MissingAttribute",
```

File: assets/queries/dockerCompose/networks_not_set/test/positive_expected_result.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[
+    {
+      "queryName": "Networks Not Set",
+      "severity": "MEDIUM",
+      "line": 7,
+      "filename": "positive1.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/networks_not_set/test/positive_expected_result.json:6]:
```suggestion
      "fileName": "positive1.yaml"
```

File: assets/queries/dockerCompose/networks_not_set/test/positive_expected_result.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    {
+      "queryName": "Networks Not Set",
+      "severity": "MEDIUM",
+      "line": 16,
+      "filename": "positive2.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/networks_not_set/test/positive_expected_result.json:12]:
```suggestion
      "fileName": "positive2.yaml"
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Comment by pgassmann** [comment]
@cxAndreFelicidade I don't understand the need for this check. Docker compose set up a separate network for each compose project, used by default if no specific network is specified for a service. It won't use the global default network on the host.

> By default Compose sets up a single [network](https://docs.docker.com/engine/reference/commandline/network_create/) for your app. Each container for a service joins the default network and is both reachable by other containers on that network, and discoverable by them at a hostname identical to the container name.

https://docs.docker.com/compose/networking/

**Pull request merged**